AppTaskQueue/appscale/taskqueue/queue_manager.py

```diff
@@ -105,9 +105,11 @@ def ensure_watch(self):
                                             self._update_queues_watch)
 
   def stop(self):
-    """ Close the Celery connections if they still exist. """
+    """ Close the Celery and Postgres connections if they still exist. """
     if self.celery is not None:
       self.celery.close()
+    if self.pg_connection is not None:
+      self.pg_connection.close()
 
```

Comment:
Should this be set to None after closing?

Reply:
Good point. Now I checked - the only place where `stop` method is called is just before deletion of entire `ProjectQueueManager` object.